scripts/update-status-go.sh

```diff
@@ -28,7 +28,7 @@ if [ $# -eq 0 ]; then
 fi
 
 STATUS_GO_REPO="${STATUS_GO_REPO:=status-go}"
-STATUS_GO_OWNER="${STATUS_GO_OWNER:=status-im}"
+STATUS_GO_OWNER="${STATUS_GO_OWNER:=rasom}"
```

Comment:
Please don't merge this PR with this change.

Reply:
I was not going to merge it that way, also it can't be built at the moment